testsuite/run_sets/build_validation_migration.yml

```diff
@@ -0,0 +1,8 @@
+# This file describes the order of features in a Build Validation test suite run.
```

Comment:
If you added it here, shouldn't you need to remove these features from another test run file?

Reply:
It's fine, we had them embedded in the init_clients features, on the scenarios removed.

Reply:
I found this description in nearly every .`yml` file for the build validation so I just copied it over to be consistent.